model-optimizer/mo/front_ng/extractor.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
Please, update the PR description using the MO PR template.

Reply:
Updated, used PR 5347 as template